addons/crm/static/src/views/forecast_kanban/forecast_kanban_column_quick_create.js

```diff
@@ -9,7 +9,7 @@ export class ForecastKanbanColumnQuickCreate extends KanbanColumnQuickCreate {
     get relatedFieldName() {
         const { granularity = "month" } = this.props.groupByField;
         const { description } = INTERVAL_OPTIONS[granularity];
-        return _t("Add next %s", description.toLocaleLowerCase());
+        return _t("next %s", description.toLocaleLowerCase());
```

Comment:
This PR may only address the issue related to the label "Add add next brol". I'm uncertain whether using `Add <t t-out="relatedFieldName"/>` directly in the component's template will result in proper translations across all languages, since translators might not be able to reorder the words as they can with the placeholder used in the `_t` function.

We might need to remove "Add" from the component template and instead include it as part of the placeholder passed to `_t`.